{"name": "test_case[case2]", "status": "broken", "statusDetails": {"message": "TypeError: 'tuple' object is not callable", "trace": "self = <test_case.Test_Case object at 0x00000255DA2AD160>\ncase = {'case_description': '/topic_collect/collect 收藏主题', 'case_expect': '{\"success\":false}', 'case_method': 'post', 'case_params': '', ...}\n\n    @pytest.mark.parametrize('case',Excel_Handler().get_excel_data())\n    def test_case(self,case):\n        print(11111111111111111,case)\n>       response = RequestHandler(case).get_response()\nE       TypeError: 'tuple' object is not callable\n\nscripts\\test_case.py:14: TypeError"}, "parameters": [{"name": "case", "value": "{'case_project': 'cnodejs项目', 'case_description': '/topic_collect/collect 收藏主题', 'case_url': 'https://cnodejs.org/api/v1/topic_collect/collect', 'case_method': 'post', 'case_params': '', 'case_expect': '{\"success\":false}', 'other': '咱们没有用户 accessToken，所以返回为false就算对'}"}], "start": 1574774793662, "stop": 1574774794115, "uuid": "2eb80e1c-53d3-4cbc-ac8f-87045666c570", "historyId": "0e4e30a8f14adab4a48c883e54d5370d", "testCaseId": "89a27520489a549f0caeaffa7948244d", "fullName": "scripts.test_case.Test_Case#test_case", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_case"}, {"name": "subSuite", "value": "Test_Case"}, {"name": "host", "value": "IG6I7Y1F5I7TZH5"}, {"name": "thread", "value": "15128-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_case"}]}